db/schema.v0.sql

```diff
@@ -434,6 +434,19 @@ CREATE TABLE IF NOT EXISTS `application` (
   PRIMARY KEY (`id`)
 );
 
+-- -----------------------------------------------------
+-- Table `ical_key`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `ical_key` (
+  `key` CHAR(36) CHARACTER SET ascii NOT NULL,
+  `requester` CHAR(255) NOT NULL,
+  `name` CHAR(255) NOT NULL,
+  `type` ENUM('team', 'user') NOT NULL,
+  `time_created` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`requester`, `name`, `type`),
+  INDEX `key_idx` (`KEY`)
+)
```

Comment:
Looks like you might be missing a semicolon here which is perhaps why so many of the tests are now failing since this could cause errors in the database setup.

Reply:
Thank you! Such a life saver